app/styles/ember-power-calendar.scss

```diff
@@ -1,3 +1,6 @@
+$cell-spacing: 2px;
```

Comment:
true but it is fixing an existing defect that I messaged to you about ... when days are not visible you start to have layout problems with the old styling